pkgs/development/python-modules/pem/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, pytestCheckHook
+, certifi
+, cryptography
+, pretend
+, pyopenssl
+, twisted
+}:
+
+buildPythonPackage rec {
+  pname = "pem";
+  version = "21.2.0";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "hynek";
+    repo = pname;
+    rev = version;
+    hash = "sha256-mftLdgtgb5J4zwsb1F/4v4K7XTy4VSZBMy3zPV2f1uA=";
+  };
+
+  patches = [  ];
+
+  postPatch = '''';
+
+  propagatedNativeBuildInputs = [];
```

Comment:
Are these empty values necessary?

Reply:
whoops. slipped in while I was reshuffling PRs. fixed.